src/code/modules/tutorial.json

```diff
@@ -71,6 +71,113 @@
           ]
         }
       ]
+    },
+    {
+      "level": 2,
+      "missions": [
+        {
+          "mission": 0,
+          "challenges":[
+            {
+              "challenge": 0,
+              "steps":[
+                {
+                  "element": "organism-color-text",
+                  "text": "Drake Color",
+                  "more": "This is the breeder's name for the drake color."
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "level": 3,
+      "missions": [
+        {
+          "mission": 1,
+          "challenges":[
+            {
+              "challenge": 0,
+              "steps":[
+                {
+                  "element": "mother",
+                  "text": "Select Alleles",
+                  "more": "Set the parent’s alleles so their baby drake will match the Target Drake."
+                },
+                {
+                  "element": "breed-button",
+                  "text": "Breed Button",
+                  "more": "Click Breed to produce offspring. Then click a baby drake that matches the target!"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "level": 3,
+      "missions": [
+        {
+          "mission": 2,
+          "challenges":[
+            {
+              "challenge": 0,
+              "steps":[
+                {
+                  "element": "mother",
+                  "text": "Select Alleles",
+                  "more": "Set the parent’s alleles so that they can produce both Target Offspring."
+                },
+                {
+                  "element": "breed-button",
+                  "text": "Breed Button",
+                  "more": "Click Breed to test if these parents can make both offspring."
+                },
+                {
+                  "element": "submit-parents-button",
+                  "text": "Submit Parents Button",
+                  "more": "Click Submit Parents when you think you have the parents that can produce BOTH drake offspring!"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "level": 4,
+      "missions": [
+        {
+          "mission": 0,
+          "challenges":[
+            {
+              "challenge": 0,
+              "steps":[
+                {
+                  "element": "father",
+                  "text": "Parent Drake",
+                  "more": "This drake’s alleles are invisible!",
```

Comment:
I would change "invisible" to "hidden". The former makes it seems like a property of the drake, as though it's a stealth drake.